site/content/en/docs/formats/yolo8.md

```diff
@@ -0,0 +1,324 @@
+---
+title: 'YOLO8'
+linkTitle: 'YOLO8'
+description: ''
+---
+
+## Format specification
+
+The YOLO8 format family allows you to define the dataset root directory, the relative paths
+to training/validation/testing image directories or *.txt files containing image paths,
+and a dictionary of class names.
+
+Family consists of four formats:
+- [Detection](https://docs.ultralytics.com/datasets/detect/)
+- [Oriented bounding Box](https://docs.ultralytics.com/datasets/obb/)
+- [Segmentation](https://docs.ultralytics.com/datasets/segment/)
+- [Pose](https://docs.ultralytics.com/datasets/pose/)
+
+Supported annotation types and formats:
+- `Bbox`
+  - Detection (only not rotated)
+  - Oriented Bounding Box,
+- `Polygon`
+  - Segmentation
+- `Skeleton`
+  - Pose
+
+The format supports arbitrary subset names, except `classes`, `names`, `backup`, `path`, `kpt_shape`, `flip_idx`.
+
+> Note, that by default, the YOLO framework does not expect any subset names,
+  except `train` and `valid`, Datumaro supports this as an extension.
+  If there is no subset separation in a project, the data
+  will be saved in the `train` subset.
+
+## Import YOLO8 dataset
+To create a Datumaro project with a YOLO8 source, use the following commands:
+
+```bash
+datum create
+datum import --format yolo8 <path/to/dataset> # for Detection dataset
+datum import --format yolo8_oriented_boxes <path/to/dataset> # for Oriented Bounding Box dataset
+datum import --format yolo8_segmentation <path/to/dataset> # for Segmentation dataset
+datum import --format yolo8_pose <path/to/dataset> # for Pose dataset
+```
+
+The YOLO8 dataset directory should have the following structure:
+
+```bash
+└─ yolo_dataset/
+   │ # a list of non-format labels (optional)  # file with list of classes
+   ├── data.yaml    # file with dataset information
+   ├── train.txt    # list of image paths in train subset [Optional]
+   ├── val.txt    # list of image paths in valid subset [Optional]
+   │
+   ├── images/
+   │   ├── train/  # directory with images for train subset
+   │   │    ├── image1.jpg
+   │   │    ├── image2.jpg
+   │   │    ├── image3.jpg
+   │   │    └── ...
+   │   ├── valid/  # directory with images for validation subset
```

Comment:
val?

Reply:
No, data.yaml says where are the images of a subset, and the folder does not have to have the same name as a subset. Tried to write it in docs in a more understandable way.

Reply:
The standard name for validation subset is "val". Nobody calls it "valid" as far as I know. @zhiltsov-max , do you have any opinion regarding that?